cpp-ch/local-engine/Parser/AggregateFunctionParser.cpp

```diff
@@ -25,6 +25,7 @@
 #include <Parser/TypeParser.h>
 #include <Common/CHUtil.h>
 #include <Common/Exception.h>
+#include <Common/logger_useful.h>
```

Comment:
should we chanage this CPP?

Reply:
Yes it is needed. otherwise LOG_INFO is unknown symbol.